docs/sql-data-sources-csv.md

```diff
@@ -201,7 +201,7 @@ Data source options of CSV can be set via:
   <tr>
     <td><code>multiLine</code></td>
     <td>false</td>
-    <td>Parse one record, which may span multiple lines, per file.</td>
+    <td>Parse one record, which may span multiple lines, per file. This option is no-op in the CSV built-in functions.</td>
```

Comment:
How about is to write in straightforward way: CSV built-in functions ignore this option.

Reply:
Sure